compiler/rustc_ast/src/ast.rs

```diff
@@ -1548,7 +1548,10 @@ pub struct QSelf {
 #[derive(Clone, Copy, PartialEq, Encodable, Decodable, Debug, HashStable_Generic)]
 pub enum CaptureBy {
     /// `move |x| y + x`.
-    Value,
+    Value {
+        /// The span of the `move` keyword.
+        move_kw: Span,
```

Comment:
The AST visitors need to visit this span.

Reply:
I'm not entirely sure I understand what that means, but I'm gonna guess that adding this to the `rustc_ast::visit::Visitor` would be enough? Correct me if I'm wrong please, this is the first time I touch rustc code (directly)😅 
```rust
    fn visit_capture_by(&mut self, capture_by: &'ast CaptureBy) {
        // Nothing to do
    }
```

Reply:
The default implementation of `visit_capture_by` in `MutVisitor` should call `visit_span` on `move_kw`.